spec/mailers/provider_mailer_spec.rb

```diff
@@ -46,39 +46,90 @@
                     'sign in path' => '/provider/sign-in')
   end
 
+    #
+    # describe 'Send provider decision chaser email' do
```

Comment:
Is there a reason to leave this comment in? looks like it was just commented out so it didn't run

Reply:
There is not. Thanks for that